src/components/shared/Project/SpendingStats.tsx

```diff
@@ -0,0 +1,104 @@
+import { Trans } from '@lingui/macro'
```

Comment:
Move this stuff into its own component, reused across v1/v2

Reply:
Nice one